src/freenet/node/DatabaseKey.java

```diff
@@ -11,13 +11,19 @@
 import freenet.support.api.Bucket;
 
 public class DatabaseKey {
+
+    private static final byte[] PLUGIN = "PLUGIN".getBytes(StandardCharsets.UTF_8);
+    private static final byte[] CLIENT_LAYER = "CLIENT".getBytes(StandardCharsets.UTF_8);
     
     private final byte[] databaseKey;
-    private final Random random;
-    
-    DatabaseKey(byte[] key, Random random) {
+
+    @Deprecated
+    DatabaseKey(byte[] key, Random unused) {
+        this(key);
+    }
```

Comment:
`random` field was not used here.
Constructor is package-local.
I marked this as `@Deprecated` for the case if somebody else is using this.